ajax/libs/ui-leaflet/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "ui-leaflet",
+  "description": "ui-leaflet - An AngularJS directive to easily interact with Leaflet maps",
+  "version": "2.0.0",
+  "filename": "ui-leaflet.min.js",
+  "author": "https://github.com/angular-ui/ui-leaflet/graphs/contributors",
+  "homepage": "https://angular-ui.github.io/ui-leaflet/",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/angular-ui/ui-leaflet.git"
+  },
+  "keywords": [
+    "angularjs",
+    "javascript",
+    "directive",
+    "leaflet",
+    "angular-ui",
+    "cli"
+  ],
+  "license": "MIT",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/angular-ui/ui-leaflet.git",
+    "fileMap": [
+      {
+        "basePath": "dist",
+        "files": [
+          "*leaflet!(_dev_mapped)*"
```

Comment:
I think `*leaflet!(*_dev_mapped*)` is better, because this can also exclude the `_dev_mapped` in some of old versions.